actors/miner/src/lib.rs

```diff
@@ -3300,6 +3300,12 @@ impl Actor {
                 if info.beneficiary != info.owner {
                     // remaining_quota always zero and positive
                     let remaining_quota = info.beneficiary_term.available(rt.curr_epoch());
+                    if remaining_quota.is_zero() {
+                        return Err(actor_error!(
+                            illegal_state,
```

Comment:
Commented on the FIP, we should pick a better code

Reply:
should use this check for both owner-beneficiary and non-owner-beneficiary.  keep both behavior the same